docs/cert-compat.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: page
+title: Certificate Compatibility
+permalink: /docs/certificate-compatibility/
+top_graphic: 1
+---
+
+[<- Back to Documentation List](/docs/)
+
+This page contains lists of software which are known to be able or unable to validate Let's Encrpyt certificates.
```

Comment:
*Encrypt

How about:

Let's Encrypt aims to be compatible with as much software as possible without compromising security. The main determining factor for whether a platform can validate Let's Encrypt certificates is whether that platform includes IdenTrust's DST Root X3 certificate in its trust store. A secondary factor is whether the platform supports modern SHA-2 certificates, since all Let's Encrypt certificates use SHA-2.

If your certificate validates on some of the "Known Compatible" platforms but not others, the problem may be a web server misconfiguration, most likely failure to provide the correct certificate chain. Test your site with [SSL Labs' Server Test](https://www.ssllabs.com/ssltest/). If that doesn't identify the problem, ask for help in our [Community Forums](https://community.letsencrypt.org/).